client/src/components/Badges/Badge.js

```diff
@@ -0,0 +1,15 @@
+import styled from '@emotion/styled';
+
+const Badge = styled.div`
+  height: 20px;
+  background-color: ${props => props.theme.primary};
+  color: ${props => props.theme.secondary};
+  border: 1px solid ${props => props.theme.third};
```

Comment:
I have the same problem.
Sometimes i destructure, sometimes not.
I think we should work consistent :/